src/test/java/com/google/api/codegen/testdata/nodejs_test_library.baseline

```diff
@@ -51,6 +54,29 @@ describe('LibraryServiceClient', function() {
         done();
       });
     });
+
+    it('invokes createShelf with error', function(done) {
+      var client = libraryV1.libraryServiceClient();
+      // Mock request
+      var shelf = {};
+      var request = {
+          shelf : shelf
+      };
+
+      // Mock Grpc layer
+      client._createShelf = function(actualRequest, options, callback) {
```

Comment:
I was trying to figure out which of the two were better as well. I went with mine to try and avoid this:

```js
// Looks a little ugly.
client._method = mockGrpcMethod(expectedRequest, null, error);
```

Maybe @jmuk can shed the light on whats best.